CHANGES.rst

```diff
@@ -1,3 +1,9 @@
+Release 0.0.8
+=============
+
+* **MinHashEncoder**: Added a "handle_missing" attribute to allow encoding
+  with missing values.
+
```

Comment:
I think that I released 0.0.7 after you added this, hence it should be added to the 0.0.7.

Reply:
Yes, I wasn't too sure about that. I've fixed it.